### ISSUE TITLE
User facing osc-specific parameters are not exposed to MIDI

### ISSUE THREAD

**Issue by ggsubs** [korginc/logue-sdk#32]
User facing osc-specific parameters are not transmitted to MIDI and either changeable with control change messages. Given Shape and ShiftShape is sent/received from MIDI, developer expectations are to include the user messages too.

**Comment by john-k-walton** [comment]
It could be useful to have six/eight midi CC's to control osc params outboard instead of diving menus. Exposing EG and LFO controls directly during performances is a noble feature,

**Comment by dukesrg** [comment]
Minilogue XD have NRPMs for all six osc parameters. I wonder why it is not implemented in NTS-1.

**Comment by leandrob13** [comment]
Crossing our fingers so that user osc params get MIDI control in 2024.

**Comment by john-k-walton** [comment]
yeah, I've given up on getting any Prologue functional updates. I use a Yorick LFE; a low frequency external modulator with LFO's, EG's, sending note velocity back to Shape, or Shift-Shape, and rewrite my code to put the best stuff on those controls. much of the stuff I want is actually in there already, I just have to use the (freely available) memory map for the all the parameters to access them for this firmware.
